test/suggestion-builder.js

```diff
@@ -0,0 +1,63 @@
+import test from 'tape'
+import suggestionBuilder from '../src/suggestion-builder'
```

Comment:
Could you add a few more test cases here?
- multiple chained suggests
- additional options passed to suggest, especially nested options like [direct generators](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters.html#_direct_generators)
- empty or missing `options` argument

Reply:
Can do! Regarding the `empty or missing options argument`, I think `text` is required on a suggester. Not 100% how we want to handle this. I would throw an error or warning if text was missing, but I don't think we do that anywhere else in the package.

 In TS text is a required arg, but for JS devs is building an invalid suggest clause, and letting it be the developers responsibility to detect/correct the right move here?

Reply:
I think the function will throw an error anyway since we try to destructure name and text out of options.

As it turns out, `text` isn't required. It can be supplied outside the context of a suggest clause as long as you supply a global text ([see docs here](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters.html#global-suggest)). I think we could support this by allowing a suggest clause to be created with empty options. What do you think then of supplying a default value for `options = {}` to allow creating a suggest clause without text?

Reply:
Global context solve that problem! I'll make text optional, and set the default object! Good call